src/parsing.rs

```diff
@@ -37,6 +37,7 @@ enum ControlOperator {
 }
 
 pub const SCOPE_MARKER: &'static str = "_CDDL_CODEGEN_SCOPE_MARKER_";
+pub const EXTERN_MARKER: &'static str = "_CDDL_CODEGEN_EXTERN_TYPE_";
```

Comment:
Can we add a test that uses this?

Reply:
I added a test for the `core_with_wasm` and `core_no_wasm` tests so that should test both core and wasm crates.